tests/external_executor/external_executor.py

```diff
@@ -38,10 +38,16 @@
 
 
 @contextlib.contextmanager
-def wrap_tx(stub):
-    stub.StartTx(Empty())
-    yield stub
-    stub.EndTx(KV.ResponseDescription())
+def wrap_tx(stub, primary):
+    with primary.client(connection_timeout=0.1) as c:
+        try:
+            c.get("/placeholder", timeout=0.1)
```

Comment:
Was this for debugging? Don't think there is an endpoint like this?

Reply:
This call is still necessary. The executor can't do any KV `put`s or `get`s unless the node it's speaking to thinks that executor is currently processing a request. So to retain the existing tests where we probe the KV API directly (ie - outside of trying to implement a _real_ executor), we need to send a dummy request to the node (this one), such that the `StartTx()` we call after returns a real `RequestDescription`.

This is another temporary situation - this low level test may not be needed shortly, or may be moved to a separate test that's just testing gRPC access on another app. I'll add a comment in the code explaining this.